ambry-store/src/main/java/com/github/ambry/store/IndexValue.java

```diff
@@ -77,7 +77,7 @@
           + ORIGINAL_MESSAGE_OFFSET_SIZE_IN_BYTES + OPERATION_TIME_SECS_SIZE_IN_BYTES + ACCOUNT_ID_SIZE_IN_BYTES
           + CONTAINER_ID_SIZE_IN_BYTES;
 
-  final static int INDEX_VALUE_SIZE_IN_BYTES_V3 = INDEX_VALUE_SIZE_IN_BYTES_V1_V2 + LIFE_VERSION_SIZE_IN_BYTES;
+  final static int INDEX_VALUE_SIZE_IN_BYTES_V3_V4 = INDEX_VALUE_SIZE_IN_BYTES_V1_V2 + LIFE_VERSION_SIZE_IN_BYTES;
```

Comment:
Does V3_V4 mean this size applies to both?

Reply:
Yes, it applies to both. Version 4 doesn't change `IndexValue` format but `IndexSegment` format only. It adds reset key life version at the beginning of index segment (together with reset key and reset key type) without touching index entries.